rio/srcrobot/constants.py

```diff
@@ -27,22 +27,21 @@ class Swerve:
 
         # Drivetrain Constants
         trackWidth = Units.inchesToMeters(26.0)
-        wheelBase = Units.inchesToMeters(26.0)
+        wheelBase = Units.inchesToMeters(31.125)
```

Comment:
This is not the correct way to configure a non-square swerve. The SwerveDriveKinematics constructor parameter list states:
moduleTranslationsMeters The locations of the modules relative to the physical center of the robot.

Here we are essentially telling a lie about the position of the modules relative to the center of the robot. The kinematics will believe us and its calculation will be wrong for where the modules really are. If you do not want to mess with this for Saturday, you could go back to a 26 inch wheel base and rotate off center.

Reply:
My thought was that we know the positions of the back two modules are at the corners, so the other two would just be 26 inches up from that. Would that not give the module positions relative to the center where the front two are less and back two are more?

Reply:
I rolled it back to 26x26 for Saturday purposes, but we need to look more into this.

Reply:
I agree we need to look into this some more.

I may have some signs wrong here, but what about somthing like the following. BTW, our current signs still confuse me, but I did not change them.
```
    trackWidth = Units.inchesToMeters(26.0)
    wheelBase = Units.inchesToMeters(26.0)
    rotationBase = Units.inchesToMeters(31.125)
    frontBaseOffset = rotationBase - wheelBase
    frontBase = (rotationBase / 2.0) - frontBaseOffset
    frontLeftLocation = Translation2d(-frontBase, -trackWidth / 2.0)
    frontRightLocation = Translation2d(-frontBase, trackWidth / 2.0)
    backLeftLocation = Translation2d(rotationBase / 2.0, -trackWidth / 2.0)
    backRightLocation = Translation2d(rotationBase / 2.0, trackWidth / 2.0)
```

Reply:
BTW, probably no need to change this now for Saturday.